### PR TITLE
Close CPPL-214

### PR DESCRIPTION
**Pull request by sorbeee**

### CODE REVIEW

**Review by VasylDobrovolskyi — APPROVED** [review]

File: src/Client/View/MainPage/DailyList/DailyListSubPage/DailyListSubPage.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+	product->set_number(idx + 1);
+	set_list_size(get_list_size() + 1);
+	UpdateNumbers(idx + 1);
+}
+
+void DailyListSubPage::set_list_size(int size)
+{
+	m_list_size = size;
+	//m_ui->ListSize->setText(QString("%1").arg(m_list_size));
```
Comment (iodream) [inline src/Client/View/MainPage/DailyList/DailyListSubPage/DailyListSubPage.cpp:41]: remove if these lines are no longer necessary

**Review by iodream — APPROVED** [review]

File: src/Client/View/Constants.h [hunk 2]
```diff
@@ -26,6 +26,8 @@ enum class MainSubPages
 	VIEW_INCOME,
 
 	SETTINGS,
+
+	DAILY_LIST,
```
Comment (yaroslav-kernitskyi) [inline src/Client/View/Constants.h:30]: Be careful with order of elements in this enum when merging

File: src/Client/View/MainPage/DailyList/DailyListSubPage/DailyListSubPage.cpp [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	for (auto it = products.begin(); it != products.end(); it++) {
+		ProductItem* item = new ProductItem(*it);
+		item->Update();
+		AppendProduct(item);
+	}
+}
+
+ProductItem* DailyListSubPage::SafeGetProduct(int idx)
+{
+	QLayoutItem *layout = m_ui->ItemsLayout_4->itemAt(idx);
```
Comment (yaroslav-kernitskyi) [inline src/Client/View/MainPage/DailyList/DailyListSubPage/DailyListSubPage.cpp:72]: Maybe rename this layout?

**Review by yaroslav-kernitskyi — APPROVED** [review]
Looks alright

**Pull request merged**